MaxiOps/CCTP_Bridge/receive/claimArbiUSDC.json

```diff
@@ -0,0 +1,32 @@
+{
+  "version": "1.0",
+  "chainId": "42161",
+  "createdAt": 1715193245588,
+  "meta": {
+    "name": "Transactions Batch",
+    "description": "",
+    "txBuilderVersion": "1.16.5",
+    "createdFromSafeAddress": "0xc38c5f97B34E175FFd35407fc91a937300E33860",
+    "createdFromOwnerAddress": "",
+    "checksum": "0x383bbde1a0fa18089fef4d0f07fca15a785b14563241a3e34530459e2fdddc97"
+  },
+  "transactions": [
+    {
+      "to": "0xC30362313FBBA5cf9163F0bb16a0e01f01A896ca",
+      "value": "0",
+      "data": null,
+      "contractMethod": {
+        "inputs": [
+          { "name": "message", "type": "bytes", "internalType": "bytes" },
+          { "name": "attestation", "type": "bytes", "internalType": "bytes" }
+        ],
+        "name": "receiveMessage",
+        "payable": false
+      },
+      "contractInputsValues": {
+        "message": "0x000000000000000000000003000000000000D6E2000000000000000000000000BD3FA81B58BA92A82136038B25ADEC7066AF315500000000000000000000000019330D10D9CC8751218EAF51E8885D058642E08A000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000A0B86991C6218B36C1D19D4A2E9EB0CE3606EB48000000000000000000000000C38C5F97B34E175FFD35407FC91A937300E3386000000000000000000000000000000000000000000000000000000000BEBC2000000000000000000000000000C38C5F97B34E175FFD35407FC91A937300E33860",
+        "attestation": "0xaa6fd30f1a8b66ad584d31d2c5db9b7094a417981a92789baef99467877d7efe0f7791fff2a0ef0f10da34ade24543df1d55375458da75b6bc8a7db0aa099ef41ba462bb0a93541c726b3bcfe719bb272aa4faebd3937881fbfe81c65ea319eec27d0b3eb0a28bd07e769351033d898bcd769c4b057761c79448403b3f4ce079bc1c"
```

Comment:
To get the attestation, we need to generate the keccek256 hash of the message and then send that to the circle api.

To get a keccek: Go [Here](https://emn178.github.io/online-tools/keccak_256.html). Paste in the message without the 0x and make sure input type is hex:
<img width="959" alt="image" src="https://github.com/BalancerMaxis/multisig-ops/assets/11885129/928f2f52-ba19-4aba-a298-143c3864d9db">
Now take the resulting hash, put a 0x infront and use it to query the circle API like so:

https://iris-api.circle.com/v1/attestations/0xecfeab2941644be44282342220f70679f423605b1d0c6dc1a5cabe37b054c61c

The response will look like this:
`{"attestation":"0xaa6fd30f1a8b66ad584d31d2c5db9b7094a417981a92789baef99467877d7efe0f7791fff2a0ef0f10da34ade24543df1d55375458da75b6bc8a7db0aa099ef41ba462bb0a93541c726b3bcfe719bb272aa4faebd3937881fbfe81c65ea319eec27d0b3eb0a28bd07e769351033d898bcd769c4b057761c79448403b3f4ce079bc1c","status":"complete"}`

You can copy pasta the attestation and siumulate/submit the tx.